src/Output.js

```diff
@@ -34,6 +34,7 @@ module.exports = class extends ChainedMap {
       'strictModuleExceptionHandling',
       'umdNamedDefine',
       'webassemblyModuleFilename',
+      'futureEmitAssets',
```

Comment:
Could you add this new entry after `'filename'`, to maintain alphabetical order?